docs/guide/dev/integration/angular/README.md

```diff
@@ -0,0 +1,245 @@
+- [divarea plugin](https://ckeditor.com/cke4/addon/divarea) must be included in your editor build, but there is no need to list it in [config.plugins](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-plugins) or [config.extraPlugins](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-extraPlugins).
+
+## Integration with ngModel
+
+The component implements the [`ControlValueAccessor` interface](https://angular.io/api/forms/ControlValueAccessor) and works with `ngModel`.
+
+To use it, first create model in your component:
+
+``` typescript
+@Component( {
+    ...
+} )
+export class MyComponent {
+    public model = {
+        editorData: '<p>Hello, world!</p>'
+    };
+    ...
+}
+```
+
+Then you can use the model in the template to enable a two-way binding.
+
+```html
+<ckeditor [(ngModel)]="model.editorData"></ckeditor>
+```
+
+## Supported `@Input` properties
+
+The following `@Input` properties are supported by CKEditor 4 Angular component:
+
+### `editorUrl`
+see [Customising CKEditor Preset or Version](#customizing-ckeditor-preset-or-version)
+
+### `config`
+
+Custom configuration can be passed to the editor via `config` attribute passed to `<ckeditor>` in component template. The following example shows {@link guide/dev/features/toolbar/README how to change the contents of the toolbar}:
+
+```html
+<ckeditor
+	data="<p>Editor' content</p>"
+	[config]="{ { toolbar: [ [ 'Bold' ] ] } }"
+></ckeditor>
+```
+
+{@linkapi CKEDITOR.config All configuration} options can be changed this way.
+
+### `data`
+
+The initial data of the editor. It can be a static value:
+
+```html
+<ckeditor data="<p>Hello, world!</p>" ...></ckeditor>
+```
+
+or a shared parent component’s property
+
+```typescript
+@Component( {
+    ...
+} )
+export class MyComponent {
+    public editorData = '<p>Hello, world!</p>';
+    ...
+}
+```
+
+```html
+<ckeditor [data]="editorData" ...></ckeditor>
+```
+
+### `tagName`
+
+Specifies the tag name of the HTML element on which the editor will be created.
+
+The default tag is `<textArea>`.
+``` html
+<ckeditor tagName="textarea" ...></ckeditor>
+```
+
+### `readOnly`
+
+Editor can be set to {@link guide/dev/features/readonly/README read-only mode} via `readOnly` property:
+
+```html
+<ckeditor
+	data="<p>Editor's content</p>"
+	readOnly="true"
+><ckeditor>
+```
+
+This property takes precedence over {@linkapi CKEDITOR.config#readOnly `config.readOnly`} setting.
+
+## Supported `@Output` properties
+
+The following `@Input` properties are supported by CKEditor 4 Angular component:
+
+### `ready`
+
+Fires when the editor is ready. It corresponds with the `editor#instanceReady` https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-instanceReady event.
+
+### `change`
+
+Fires when the content of the editor has changed. In contrast to `change` - only emits when data really changed thus can be successfully used with `[data]` and two way `[(data)]` binding.
```

Comment:
"Fires when the content of the editor has changed." is duplicated **three** times, this makes it extremely redundant.

Reply:
How `change` event can work "In contrast to `change` "? `change` is the event that is described here.